InnerEye/ML/dataset/full_image_dataset.py

```diff
@@ -262,6 +264,40 @@ def _load_dataset_sources(self) -> Dict[str, PatientDatasetSource]:
                                     )
 
 
+def converts_channels_to_file_paths(channels: List[str],
```

Comment:
naming conventions would suggest to call that "convert..." rather than "converts...."

Reply:
Changed naming conventions from "converts..." to "convert..."